constph.py

```diff
@@ -55,9 +55,17 @@
 import simtk.unit as units
 import pymbar
 
+def strip_unit(quant):
+    """Strips the unit from a simtk.units.Quantity object and returns it's value in OpenMM consistent units.
+        Returns quant if object is not a quantity."""
+    if units.is_quantity(quant):
+        return quant.in_unit_system(units.md_unit_system)._value
+    else:
+        return quant
+
 # MODULE CONSTANTS
 kB = units.BOLTZMANN_CONSTANT_kB * units.AVOGADRO_CONSTANT_NA
-kB = kB.in_units_of(units.kilocalories_per_mole/units.kelvin)
+kB = strip_unit(kB)
```

Comment:
We don't want to strip units globally.  Please don't do this.  

We just want to avoid the unit-stripping overhead in the regions of the code that are called over and over again, which are only three lines in `setTitrationState`.  We can do this by caching the stripped values.